include/graphics.h

```diff
@@ -0,0 +1,36 @@
+/*
+This file is part of TicTacToe.
+
+TicTacToe is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+TicTacToe is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with TicTacToe.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef TICTACTOE_GRAPHICS_H
+#define TICTACTOE_GRAPHICS_H
+
+#if !defined(NXDK)
+#include <SDL2/SDL.h>
+#else
+#include <SDL.h>
+#endif
+
+int width;
```

Comment:
I find this naming a bit.... weak. It can easily confused with the width of any object that we might use later on. Maybe renaming it to `screen_width` or something else you might find better suited.